server/metrics/measurement.go

```diff
@@ -157,29 +161,29 @@ func (m *Measurement) GetAuthErrorTags(err error) map[string]string {
 }
 
 func (m *Measurement) SaveMeasurementToContext(ctx context.Context) (context.Context, error) {
-	if m.span == nil {
+	if m.datadogSpan == nil || m.jaegerSpan == nil {
```

Comment:
This will return an error if one of them is turned off right?

Reply:
Good catch! The logic needs to be AND.